chartsxblock/static/js/src/chartsxblock-studio.js

```diff
@@ -145,26 +145,27 @@ function ChartTable(targetDiv, inputJSON){
         var delete_column_row = $("<tr>"),
             delete_column = '';
 
-        for(var i = 0; i < this.columns; i++) {
-            delete_column = $('<td class="actions"><button data-columnnumber="' + i + '" class="deleteColumn fa fa-times" aria-hidden="true"></button></td>');
-            delete_column_row.append(delete_column.clone());
-        }
-        delete_column_row.append($('<td class="actions"></td>'));
-        this.div.find("tfoot").append(delete_column_row);
-
         for (var i = 0; i < this.rows; i++) {
             var row = $("<tr>"),
                 deleteButton = '<td class="actions"></td>';
             for (var j = 0; j < this.columns; j++) {
                 var field = $("<td>" + this.data[i][j] + "</td>");
                 row.append(field);
+
+                if(i == 0) {
```

Comment:
Hajde ovo samo radi stila prebaci u i == this.rows, tako da na zadnjem redu produži.